example/reflect/README.md

```diff
@@ -0,0 +1,129 @@
+# Running The Reflect Example
+
+## Ziti Network Setup
+### Create A Ziti Network
+You will need to have ziti executables available to run from your command line. If you do not have them, the easiest way to set yourself up is to run ziti express install.
+
+To run express install, check out the instructions [here](https://openziti.github.io/ziti/quickstarts/network/local-no-docker.html)
+
+If running express install, you will need to add the ziti binaries to your path which you can do with the following command.
```

Comment:
let's remove these bits about putting it on the path. Let people figure out how to run the file.